validator/src/admin_rpc_service.rs

```diff
@@ -212,6 +216,16 @@ pub trait AdminRpc {
         meta: Self::Metadata,
         pubkey_str: String,
     ) -> Result<HashMap<RpcAccountIndex, usize>>;
+
+    // This method does not itself do an accounts scan, but returns data only relevant to nodes
+    // that do support accounts-scan RPC apis
```

Comment:
Legacy comment?

Reply:
Oh it was above the call for `get_secondary_index_key_size` too when I first made the PR, must have been removed at some point. I'll remove here as well.